### PR TITLE
ACS-6880: catch node mapping errors [bulk-ingester]

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    private AlfrescoNodeMapper alfrescoNodeMapper;
+    @Mock
+    private NodePublisher nodePublisher;
+    @Mock
+    private ApplicationManager applicationManager;
+    @InjectMocks
+    private BulkIngestionProcessor bulkIngestionProcessor;
+
+    @Test
+    public void shouldPublishNodesFromGivenIdRangeAndShoutDownApplication()
```
Comment (tpage-alfresco) [inline bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionProcessorTest.java:72]:
```suggestion
    public void shouldPublishNodesFromGivenIdRangeAndShutDownApplication()
```
Although I like the idea of "shouting down" the application too.

Comment (tpage-alfresco) [reply bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionProcessorTest.java:72]: Ok - scrap that. This is a real thing I've not heard of! :facepalm:

Comment (tpage-alfresco) [reply bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionProcessorTest.java:72]: Or maybe it's just a common typo :joy:

**Review by tpage-alfresco — APPROVED** [review]
LGTM

File: bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionProcessorTest.java (hunk 1 above)
Comment (Pawel-608) [reply bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionProcessorTest.java]: fixed 😂

**Pull request merged**